src/index.tsx

```diff
@@ -1,8 +1,6 @@
 import "ts-polyfill";
 
 import { Provider } from "mobx-react";
-// https://github.com/mobxjs/mobx-react#observer-batching
-import "mobx-react/batchingForReactDom";
```

Comment:
This was added as a result of a warning generated from an earlier round of dependency updates. Now a subsequent update removes the need for it and so removing it now removes a new console warning indicating that it's no longer necessary.